test/src/com/xilinx/rapidwright/edif/TestEDIFNetlist.java

```diff
@@ -450,6 +450,38 @@ public void testBussedPortNamingCollision(@TempDir Path path) {
         Assertions.assertEquals(port1.getWidth(), testPort1.getWidth());
     }
 
+    @Test
+    public void testBussedPortReNamingCollision(@TempDir Path path) {
+        final EDIFNetlist origNetlist = EDIFTools.createNewNetlist("test");
+
+        EDIFCell top = origNetlist.getTopCell();
+
+        EDIFCellInst ff = top.createChildCellInst("ff", Design.getPrimitivesLibrary().getCell("FDRE"));
+        origNetlist.getHDIPrimitivesLibrary().addCell(ff.getCellType());
+
+        String portName = "unfortunate_name";
+
+        // Create two ports, one single-bit and another bussed with the same root name
```

Comment:
Is this comment correct? Looks like two multi-bit ports?

Reply:
Thanks, updated.